gradle-plugin/LICENSE

```diff
@@ -0,0 +1,373 @@
+Mozilla Public License Version 2.0
+==================================
```

Comment:
ISTM `gradle-plugin` doesn't need its own copy of the `LICENSE` file if one already exists in the top-level repo. Or does it?

Reply:
Yeah, this is true, but it doesn't hurt and I want the plugin to be easy to extract as a stand-alone.